library/groovy

```diff
@@ -44,39 +44,44 @@ Directory: jre9
 
 Tags: 2.5.0-alpha-1-jdk7, 2.5-jdk7
 GitFetch: refs/heads/pre-release
-GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+GitCommit: fbf90de5f7f6f527764db38a5c6152a9139ed323
 Directory: jdk7
 
 Tags: 2.5.0-alpha-1-jre7, 2.5-jre7
 GitFetch: refs/heads/pre-release
-GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+GitCommit: fbf90de5f7f6f527764db38a5c6152a9139ed323
 Directory: jre7
 
 Tags: 2.5.0-alpha-1-jdk7-alpine, 2.5-jdk7-alpine
 GitFetch: refs/heads/pre-release
-GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+GitCommit: fbf90de5f7f6f527764db38a5c6152a9139ed323
 Directory: jdk7-alpine
 
 Tags: 2.5.0-alpha-1-jre7-alpine, 2.5-jre7-alpine
 GitFetch: refs/heads/pre-release
-GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+GitCommit: fbf90de5f7f6f527764db38a5c6152a9139ed323
 Directory: jre7-alpine
 
 Tags: 2.5.0-alpha-1-jdk8, 2.5-jdk8, 2.5.0-alpha-1-jdk, 2.5-jdk
 GitFetch: refs/heads/pre-release
-GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+GitCommit: fbf90de5f7f6f527764db38a5c6152a9139ed323
 Directory: jdk8
 
 Tags: 2.5.0-alpha-1-jre8, 2.5-jre8, 2.5.0-alpha-1-jre, 2.5-jre, 2.5.0-alpha-1, 2.5
-GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+GitCommit: fbf90de5f7f6f527764db38a5c6152a9139ed323
```

Comment:
This one is missing its `GitFetch: refs/heads/pre-release`. It just happens to work since other tags fetch the same commit.